doc.html

```diff
@@ -1,1005 +1,1003 @@
 <!DOCTYPE html>
 <html>
+    <head>
+        <meta charset="utf-8">
+        <title></title>
+        <meta name="author" content="">
+        <meta name="description" content="">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <link href="css/style.css" rel="stylesheet">
+        <link href="css/fontawesome.min.css">
```

Comment:
So, not *quite* just massive whitespace changes in the first commit. 😆

Reply:
Refactored into a separate commit, thanks for pointing out!